app/models/invitation.rb

```diff
@@ -2,12 +2,18 @@ class Invitation < ApplicationRecord
   include Mumuki::Domain::Syncable
 
   belongs_to :course
+
+  validate :ensure_not_expired, on: :create
   validates_uniqueness_of :code
 
   defaults do
     self.code ||= self.class.generate_code
   end
 
+  def ensure_not_expired
```

Comment:
same here?

Reply:
In general we are using `!` when the `ensure_` explicitly raises an exception